app/scripts/modules/core/pipeline/config/triggers/travis/travisTrigger.module.ts

```diff
@@ -121,6 +121,6 @@ module(TRAVIS_TRIGGER, [
     formatLabel: (trigger: IBuildTrigger) => {
       return $q.when(`(Travis) ${trigger.master}: ${trigger.job}`);
     },
-    selectorTemplate: '<travis-trigger-options command="vm.command"></travis-trigger-options>',
+    selectorTemplate: require('./selectorTemplate.html'),
```

Comment:
Sorry, I really can't inline this. It doesn't work. It causes deck to try to load it as a url.
`https://spinnaker/%3Ctravis-trigger-options%20command=%22vm.command%22%3E%3C/travis-trigger-options%3E`